lib/sales_engine.rb

```diff
@@ -0,0 +1,25 @@
+# require 'csv'
+require_relative './customer_repository'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.